extensions/ql-vscode/src/mocks/gh-api-request.ts

```diff
@@ -12,13 +12,17 @@ export enum RequestKind {
   GetVariantAnalysisRepoResult = 'getVariantAnalysisRepoResult',
 }
 
+export interface BasicErorResponse {
+  message: string;
+}
+
 export interface GetRepoRequest {
   request: {
     kind: RequestKind.GetRepo
   },
   response: {
     status: number,
-    body: Repository
+    body: Repository | BasicErorResponse | undefined
```

Comment:
Instead of using `undefined`, we might want to make the `body` property optional. Those two should be equivalent, but I think that will be clearer when comparing the JSON files to these types since the property will be missing from the JSON files.

```suggestion
    body?: Repository | BasicErorResponse
```

Reply:
That's a good shout! Made the changes.